### PR TITLE
Create Header

### PR DESCRIPTION
**Pull request by Neil-B-Linton**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,15 @@
   <head>
     <link rel="stylesheet" href="./styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>What's for Dinner?</title>
   </head>
-  <body>
-    
+  <body background="./assets/burger-friends.jpg"alt="Burger With Friends">
```
Comment (alexmkio) [inline index.html]: add space between url and alt

File: index.html [hunk 2]
```diff
@@ -2,9 +2,15 @@
...
+    <header>
+      <h1>What's for Dinner?</h1>
+      <button>add a recipe</button>
```
Comment (alexmkio) [inline index.html:10]: let's use CSS to make uppercase

**Review by alexmkio — CHANGES_REQUESTED** [review]
Overall really good, but a couple changes I'd like to see first

File: index.html (hunk 1 above)
Comment (Neil-B-Linton) [reply index.html]: Added space between url and alt

File: index.html (hunk 2 above)
Comment (Neil-B-Linton) [reply index.html:10]: Added CSS to make button uppercase

**Review by alexmkio — APPROVED** [review]
Thanks for the changes!

**Pull request merged**